src/int/bswap.rs

```diff
@@ -0,0 +1,35 @@
+use crate::int::Int;
+
+fn bswap<I: Int + From<u8>>(x: I) -> I {
+    let mut ret = I::ZERO;
+    for (i, byte) in x.to_be_bytes().into_iter().enumerate() {
+        ret |= I::from(byte) << (i as u32 * 8);
+    }
+    ret
+}
+
+intrinsics! {
+    #[maybe_use_optimized_c_shim]
+    #[cfg(any(target_pointer_width = "32", target_pointer_width = "64"))]
+    #[inline(never)]
```

Comment:
Oh sorry, I didn't mean this line had to included - I just copied my code from godbolt where you need it to force assembly output.

Docs usually go above the `#[...]` attributes (makes no difference of course)

Implementation LGTM, thanks for the additions

Reply:
Removed the inlines, for one less useless jump.

Reply:
As for docs - they are below the attributes everywhere else (and I think they have to be, otherwise they break the macros)

Reply:
Ah, I just meant rustc style in general but you're right about the macro. Guess we just don't have docs on too many of the intrinsics.

Reply:
If there's nothing else to fix, can this be merged then? I have 2 more to go after this )

Reply:
I can’t merge, Amaneau needs to take a look.

Feel free to open the other PRs if you have an implementation ready

Reply:
Oh I see. 

They are ready but they sort of depend on each other (if only superficially). Merging any of them would require resolving conflicts with the others, so I hoped to get them in sequencially.